src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.25
 ++++++
+* vm image: support accept market terms to use vm images
```

Comment:
Recommend: Added `vm image accept-terms` command to support using certain marketplace images.